open_fdd/desktop/drivers/bacnet_driver.py

```
@@ -0,0 +1,235 @@
+                    ],
+                }
+            },
+        }
+        req = urllib.request.Request(
+            f"{base}/client_read_multiple",
+            method="POST",
+            headers=headers,
+            data=json.dumps(payload).encode("utf-8"),
+        )
+        try:
+            with urllib.request.urlopen(req, timeout=25) as resp:
```

Ruff (0.15.12)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)